lib/pharos/phases/configure_calico.rb

```diff
@@ -30,6 +30,8 @@ def call
         Pharos::Kube.apply_stack(
           @master.api_address, 'calico',
           ipv4_pool_cidr: @config.network.pod_network_cidr,
+          ipip_mode: @config.network.calico&.ipip_mode || 'Always',
```

Comment:
This is annoying, but the `.default(...)` value defined on the struct attribute is mostly useless, because the struct itself doesn't have a default value... dealing with the configs would be so much easier if the nested structs also had default values.